### ISSUE TITLE
Path to scenario folder to restore state

### ISSUE THREAD

**Issue by musamarcusso** [automl/SMAC3#366]
Hi,

I was running the example on restoring the state for the branin function and I run into this error

```
INFO:smac.scenario.scenario.Scenario:Output to restored
Output directory= restore_me/run_1
['traj_old.csv', 'scenario.txt', 'runhistory.json', 'param_config_space.pcs', 'stats.json', 'traj_aclib2.json']
Traceback (most recent call last):
  File "restore_state.py", line 77, in <module>
    main()
  File "restore_state.py", line 64, in main
    shutil.copy(traj_path, new_traj_path)
  File "/usr/lib/python3.5/shutil.py", line 235, in copy
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/usr/lib/python3.5/shutil.py", line 115, in copyfile

    with open(dst, 'wb') as fdst:
FileNotFoundError: [Errno 2] No such file or directory: 'restored/traj_aclib2.json'
```

I figure the issue comes from this line:
https://github.com/automl/SMAC3/blob/56a5af9e3666ae54f7c126b62ae44852af65842d/examples/branin/restore_state.py#L46

The output scenario folder should redefined as `output_dir = os.path.join(original_scenario.output_dir, 'run_1')`, right?

**Comment by mlindauer** [comment]
Dear Musa,

Thank you for reporting this.
Indeed, it looks like we introduced a bug by changing the output directory structure in our last release. We will look into it and fix it for the next release.

> The output scenario folder should redefined as output_dir = os.path.join(original_scenario.output_dir, 'run_1'), right?

It doesn't have to be 1 if you use another seed.

**Comment by musamarcusso** [comment]
Hi Marius, thanks for the quick response. I noticed that that number was the seed number after running some other scripts I have here. Does this differentiation mean that upon running pSMAC I can have the same output folder? Or is there some other reason for the extra folder?

**Comment by mlindauer** [comment]
Yes, if you want to run SMAC in parallel in the same folder but with different seeds, this should now be more comfortable.

**Comment by shukon** [comment]
I guess this is because the output-dir is not created in the `Scenario.__init__`, but in the SMAC-facade. I'll fix this asap.

**Comment by shukon** [comment]
fixed by #389
